Source/DafnyCore/Verifier/QuickSyntaxBasedVerifier.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Numerics;
+using Microsoft.Dafny.Triggers;
+
+namespace Microsoft.Dafny.ExprExtensions;
+
+/// <summary>
+/// This class's code contains helpers that help verifying assertions without
+/// the use of Boogie
+/// See Translator.IsExprAlways for a smaller version for Boogie expressions
+/// </summary>
+public static class QuickSyntaxBasedVerifier {
```

Comment:
You've described this verifier by saying it's "quick" "trivial" and "obvious", but I'd like to be more precise about what the scope is. And once it's clear what the scope is, is it complete or do you expect it to grow more?

Reply:
What is unclear exactly? Can you please be more precise about what do you mean by "scope"? I precisely defined the notion of trivialness in this PR's description. In heuristics, there is nothing complete. Would you like me to put the PR's description in the comments of the class? Or rename the class?